ynput/ayon-core
# Chore: Move AfterEffects client code

## Changelog Description
Moved AE client code next to server codebase.

## Additional information
Copied from `ayon_core/tools/adobe_webserver` to aftereffects addon and removed the tool from ayon core as is not used anymore.

## Testing notes:
1. Create package of aftereffects and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. AfterEffects integration should work as before.
